pkg/bundle/bundle.go

```diff
@@ -98,15 +98,6 @@ func (b *ProtobufBundle) validate() error {
 		}
 	}
 
-	// if bundle version >= v0.3, require verification material to not be X.509 certificate chain (only single certificate is allowed)
-	if semver.Compare(bundleVersion, "v0.3") >= 0 {
-		certs := b.Bundle.VerificationMaterial.GetX509CertificateChain()
-
-		if certs != nil {
-			return errors.New("verification material cannot be X.509 certificate chain (for bundle v0.3)")
-		}
-	}
```

Comment:
This is required [if your verification materials are from PGI and you're using keyless signing](https://github.com/sigstore/protobuf-specs/blob/main/protos/sigstore_bundle.proto#L91-L92), but I don't know of an easy way to test for that here...

Reply:
Could we check the certificate issuer in the leaf certificate to see if it matches `O=sigstore.dev, CN=sigstore-intermediate` and do this validation in that case?

Also, is it critical to validate this right here? If we had a trusted root from PGI and a certificate chain in the bundle, wouldn't it become clear in `VerifyLeafCertificate` that the intermediates in the bundle hadn't been issued by PGI?

Reply:
This came up when we were writing these requirements previously - I think this is more of a promise to clients that a trust root coming from PGI will meet the stated requirements, for if you're writing a client only for the public instance. I don't think we need to enforce that here.